components/brave_shields/browser/ad_block_base_service.cc

```diff
@@ -163,7 +183,6 @@ void AdBlockBaseService::OnDATFileDataReady() {
     LOG(ERROR) << "Could not obtain ad block data";
     return;
   }
-  ad_block_client_.reset(new AdBlockClient());
```

Comment:
This would remove the already added tags on the instance. Deserialize should already be resetting all members to a valid state.